### ISSUE TITLE
Tabs on edit gradeable cover up footer

### ISSUE THREAD

**Issue by sbelsk** [Submitty/Submitty#6859]
**Describe the bug**
On the create/edit gradeable page (especially when viewed on a small screen), there is extra space above the div with the content of the page, and if there is enough content displayed to fill the whole screen, that div hides the Submitty footer (either partially or fully, depends on the length of the page)
![Screenshot_10](https://user-images.githubusercontent.com/71195502/127705001-d182316a-0374-4860-bc4b-d189d5d458ab.png)

**Expected behavior**
The space above the page content div should be consistent with the rest of the pages on the site, and if the page is too long, the user should be able to scroll in order to see the footer.

**Comment by ForchaGlen** [comment]
@sbelsk Please i would love to work on this issue.

**Comment by immortalcodes** [comment]
@ForchaGlen u can work on any issue and if you make some progress then link a pull request from the branch in your forked repo.
